.github/workflows/smoke-test.yml

```diff
@@ -140,7 +140,7 @@ jobs:
           path: python/src/opendp/lib
       
       - name: Install package
-        run: python -m pip install -e .
+        run: python -m pip install -e .[scikit-learn]
```

Comment:
Reminder to self:
- #1037

so latter on we can confirm in CI that it works both with and without the optional dependencies.